.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to GLPI
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+Please note current repository is about GLPI core only. All related plugins are the responsibility of their respective owners and all requests must be done on their own systems.
+
+**Write only in english!**
+
+## Bugs
+
+If you found a bug, the first steps to do are:
+- check if you're using the latest version, and if not, upgrade to see if problem is not yet solved,
+- search in already existing tickets to see if someone else already report the issue.
+
+If the last release does not solve the issue and there is no existing ticket, create one using the bug report template. Be sure you provide all requested details, log entries, and whatever may be usefull to understand and reproduce the issue.
+
+If developpers are not able to reproduce the issue, it may not be fixed.
+
+Note that issues are handled on a best-effort basis. If you need a quick fix or any guarantee, take a look at [profesionnal services](http://services.glpi-network.com/) or [partners](https://glpi-project.org/partners/).
+
+## Features
+
+If you want to work on a new feature, open a ticket using the feature template, so we can discuss on it.
+
+If you're just asking for something to be added without doing it yourself, please condier adding (or upvoting) an enhancement on the [suggestion website](https//suggest.glpi-project.org).
+
+## Support request
+
+You have a question on GLPI usage, or want to know what are the capabilities.
```

Comment:
`?` at the end of sentence ?

Reply:
Depends if you ask a question or just tell something :D Guess both are quite the same, this do not change anything.